lib/webdriver.py

```diff
@@ -144,14 +144,18 @@ def _get_driver(self) -> Driver:
             driver_version=driver_version,
             headless=True,
             uc_cdp_events=True,
-            undetectable=True,
+            undetectable=True
         )
         logger.debug("Using browser version: %s", driver.caps["browserVersion"])
 
         driver.add_cdp_listener("Network.requestWillBeSent", self._headers_listener)
 
         logger.debug("Loading Southwest Check-In page")
-        driver.get(CHECKIN_URL)
+        driver.open(CHECKIN_URL)
+        driver.type("input[name='recordLocator']", RECORD_LOCATOR)
+        driver.type("input[name='firstName']", "fname")
+        driver.type("input[name='lastName']", "lname")
+        driver.js_click("button[role='submit']")
```

Comment:
Is there any reason `js_click` is used here over `driver.click`?

Reply:
Sometimes it's giving me an error saying click distrusted by another element. With js_click it does not happen